modules/test/playwright/tests/object-web/utils/mockObjectFieldsObjectEntry.ts

```diff
@@ -0,0 +1,362 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {getRandomInt} from '../../../utils/getRandomInt';
+import getRandomString from '../../../utils/getRandomString';
+
+type objectFieldBusinessType = (typeof objectFieldBusinessTypes)[number];
+
+type objectFieldBusinessTypesLabelName = {
+	[K in objectFieldBusinessType]: objectFieldLabelName;
+};
+
+type objectFieldLabelName = {
+	label: string;
+	name: string;
+};
+
+const objectFieldBusinessTypes = [
+	'autoIncrement',
+	'boolean',
+	'decimal',
+	'encrypted',
+	'integer',
+	'longInteger',
+	'longText',
+	'multiselectPicklist',
+	'picklist',
+	'precisionDecimal',
+	'richText',
+	'text',
+] as const;
+
+function mockObjectEntry(
+	objectFieldBusinessTypesInfo: objectFieldBusinessTypesLabelName,
+	listTypeEntries: string[]
+) {
+	return {
+		[objectFieldBusinessTypesInfo.boolean.name]: Math.random() < 0.5,
+		[objectFieldBusinessTypesInfo.decimal.name]: parseFloat(
+			Math.random().toFixed(10)
+		).toString(),
+		[objectFieldBusinessTypesInfo.encrypted.name]: getRandomString(),
+		[objectFieldBusinessTypesInfo.integer.name]: Math.floor(
+			Math.random() * 100
+		),
+		[objectFieldBusinessTypesInfo.longInteger.name]: getRandomInt(),
+		[objectFieldBusinessTypesInfo.longText.name]: getRandomString(),
+		[objectFieldBusinessTypesInfo.multiselectPicklist.name]: [
+			listTypeEntries[0],
+			listTypeEntries[1],
+		],
+		[objectFieldBusinessTypesInfo.picklist.name]: {
+			key: listTypeEntries[0],
+		},
+		[objectFieldBusinessTypesInfo.precisionDecimal.name]: parseFloat(
+			Math.random().toFixed(15)
+		).toString(),
+		[objectFieldBusinessTypesInfo.richText.name]: getRandomString(),
+		[objectFieldBusinessTypesInfo.text.name]: getRandomString(),
+	};
+}
+
+function mockObjectFields(
+	objectFieldBusinessTypesInfo: objectFieldBusinessTypesLabelName,
+	listTypeDefinition: ListTypeDefinition
+) {
+	return [
+		{
+			DBType: 'String',
+			businessType: 'AutoIncrement',
+			indexed: true,
+			indexedAsKeyword: false,
+			indexedLanguageId: 'en_US',
+			label: {
+				en_US: objectFieldBusinessTypesInfo.autoIncrement.label,
+			},
+			localized: false,
+			name: objectFieldBusinessTypesInfo.autoIncrement.name,
+			objectFieldSettings: [
+				{
+					name: 'initialValue',
+					value: '1',
+				},
+			],
+			readOnly: 'false',
+			readOnlyConditionExpression: '',
+			required: false,
+			state: false,
+			system: false,
+			type: 'String',
+			unique: false,
+		},
```

Comment:
Hey @igor-franca, I think we can use more logic here instead of repeating this code a lot, I'm thinking about:

* Create an object that will contain all the information about the DBType and businessType of each field:

```ts
const businessTypeInfo = {
	autoIncrement: {
		DBType: 'String',
		businessType: 'AutoIncrement'
	}
	
	...
  }
```
* Then we would create a new function to generate this object field for us:

```ts
function createObjectField(
	businessType: keyof ObjectFieldBusinessTypesLabelName,
	objectFieldBusinessTypesInfo: ObjectFieldBusinessTypesLabelName,
	additionalSettings: Partial<ObjectField> = {}
  ): ObjectField {
	const baseField = {
	  indexed: true,
	  indexedAsKeyword: false,
	  indexedLanguageId: '',
	  localized: false,
	  objectFieldSettings: [],
	  readOnly: 'false',
	  readOnlyConditionExpression: '',
	  required: false,
	  state: false,
	  system: false,
	  unique: false,
	};
  
	return {
	  DBType: businessTypeInfo[businessType].DBType,
	  businessType: businessTypeInfo[businessType].businessType,
	  label: {
		en_US: objectFieldBusinessTypesInfo[businessType].label,
	  },
	  name: objectFieldBusinessTypesInfo[businessType].name,
	  type: businessTypeInfo[businessType].DBType,
	  ...baseField,
	  ...additionalSettings,
	};
  }
```
* And finally we would use this createObjectField function in mockObjectFields:

```ts
function mockObjectFields(
	objectFieldBusinessTypesInfo: ObjectFieldBusinessTypesLabelName,
	listTypeDefinition: ListTypeDefinition
) {
	return [
		createObjectField('autoIncrement', objectFieldBusinessTypesInfo, {
			objectFieldSettings: [{ name: 'initialValue', value: '1' }],
			indexedLanguageId: 'en_US',
		  }),

		  ...
	];
}		  
```

Of course you can name the functions and variables whatever name you think is best, this is just a suggestion